roles/pre_tasks/README.md

```diff
@@ -13,6 +13,9 @@ Available variables are listed below, along with default values defined (see def
 ```yaml
 tower_working_location: "/root/"
 
+# Use the default tower installation template
+pre_tasks_process_template: True
```

Comment:
We should use `true` without capital because it's the only variant that will be accepted in YAML 1.2 (which will come, exactly like IPv6).

Reply:
Updated, Agree with changes